setup.py

```diff
@@ -59,14 +59,15 @@ def read(fname):
     'boto >= 2.32',
     'requests',
     'boto3',
+    'google-cloud-storage',
```

Comment:
How heavy is this dependency? (download size, subdependencies, any compilation?, etc)

Reply:
The wheel is [72.7 kB on pypi](https://pypi.org/project/google-cloud-storage/#files). It has [three other GCP dependencies](https://github.com/googleapis/google-cloud-python/blob/master/storage/setup.py), I do not see any compilation required. I think optional installation for smart_open is a good idea, and can move it into `extras_require` in the setup.py

Reply:
Nah. If it's light (and reasonably stable), I'd prefer a direct dependency, to keep things simple.